container/features/src/main/resources/features-minion.xml

```diff
@@ -36,19 +36,10 @@
     </feature>
 
     <feature name="opennms-trapd-handler-minion" description="OpenNMS :: Trapd :: Handler :: Minion" version="${project.version}">
```

Comment:
Can we call this feature `opennms-trapd-listener` instead of `opennms-trapd-handler-minion` to be consistent with the Syslog implementation?